app/static/translationPackage/getStoreModule.ts

```diff
@@ -0,0 +1,5 @@
+import { language } from "./store/language"
```

Comment:
I have to admit I am confused as to why this is needed, vs the usual way we access things from the store (I believe that's what this is achieving?)

Reply:
so I can probably put this into the store/language file but essentially since we are isolating this into a package, I am imagining a function like getStoreModule that people can later use from our package to register the language module into their store

Reply:
(the full isolating into an npm package is a future ticket)